### ISSUE TITLE
PR 30 Breaks SSH

### ISSUE THREAD

**Issue by zachab4** [codekow/ocp-code-server#36]
https://github.com/codekow/ocp-code-server/pull/30 Removed the user from etc/passwd and now ssh client errors on no valid user for user ID

Log output
```shell
git clone git@github.com:me/kludge-ops.git
Cloning into 'kludge-ops'...
No user exists for uid 1000940000
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by zachab4** [comment]
Did some testing...new deployments work with #30, however for all of the previously deployed code servers that existed before #30 continue to have the above error.  There must be something in the home directory (since that persists) that is causing the conflict.  Could it be the SSH keys themselves?  Maybe on keygen or when adding to ssh-agent?

**Comment by KyWa** [comment]
I'll test this on a fresh deployment and new keys to see what occurs. I did get a report of some other interesting things with code-server 4.4.0, but not related to this (mostly on default settings when startup occurs)

**Comment by codekow** [comment]
This should be resolved in the recent pr #37
